src/vxfw/ScrollBars.zig

```diff
@@ -163,10 +294,16 @@ pub fn draw(self: *ScrollBars, ctx: vxfw.DrawContext) Allocator.Error!vxfw.Surfa
             scroll_bar.writeCell(
                 0,
                 @intCast(row),
-                vertical_scrollbar_thumb,
+                if (self.is_hovering_vertical_thumb)
+                    vertical_scrollbar_hover_thumb
```

Comment:
Use the drag style here if we are dragging

Reply:
Done in https://github.com/rockorager/libvaxis/pull/148/commits/49e1d1d0a7ca607cd68a07eb47c26018e09b13ac 🙌